app/models/alchemy/picture_version.rb

```diff
@@ -1,13 +1,19 @@
 class Alchemy::PictureVersion < ActiveRecord::Base
-  has_many :alchemy_dragonfly_signatures, class_name: 'Alchemy::DragonflySignature', foreign_key: :signature,
+  has_one :alchemy_dragonfly_signature, class_name: 'Alchemy::DragonflySignature', foreign_key: :signature,
```

Comment:
Align the indentation on the following line.

Reply:
The other side of this association also has `dependent: destroy` which I suspect it should not.